### ISSUE TITLE
Back button announced as unlabelled button in TalkBack

### ISSUE THREAD

**Issue by imgiseverything** [wix/react-native-navigation#7962]
### What happened?

In our app (Eurostar),  we are using v7.40.1 of react-native-navigation and the back button in the topBar reads out as either 'Unlabelled, button' or 'Detected icon, back, button'. It works as expected in VoiceOver/iOS and seems to be an Android only issue.

![Image](https://github.com/user-attachments/assets/b5f446d6-06c1-44e3-8205-66300f963d77)

### What was the expected behaviour?

With TalkBack turned on on Android tapping on the back button inside the topBar should announce 'Back, button'

### Was it tested on latest react-native-navigation?

- [x] I have tested this issue on the latest react-native-navigation release and it still reproduces.

### Help us reproduce this issue!

Open the Eurostar app (latest version) and find a screen with a topBar - the log-in screen is a good example.

Link: https://play.google.com/store/apps/details?id=com.eurostar.androidapp&hl=en_GB&pli=1

### In what environment did this happen?

React Native Navigation version: 7.40.1
React Native version: 0.74.6
Has Fabric (React Native's new rendering system) enabled: (yes/no) 
Node version: 20.10.0
Device model: Galaxy A21s / SM-A2117F/DSN
Android version: S(Android 12)
